jenkins/Jenkinsfile-nightly

```diff
@@ -159,6 +159,11 @@ pipeline {
           sh '''
             timeout 60  bash -x ./.jenkins-scripts/test-slurm-job.sh
           '''
+
+          echo "Test Spack (including build)"
```

Comment:
We might want to add this to the multi-node test, but it shouldn't matter much for this test so I think it is okay.